user-docs/documentation/docs/how-to-guides/3-migration-guides/1-yarn-to-pnpm.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 1
+---
+
+# From Yarn to Pnpm
+
+[Yarn](https://yarnpkg.com/) and [Pnpm](https://pnpm.io/) are both package managers.
+
+## Why?
+
+In order to choose between these two alternatives, we have checked the following criteria:
+
+- popularity: this is the package manager that is used by all the major frameworks (NextJS, VueJS, Vite, etc.)
+- performance : pnpm uses hardlinks for `node_modules` by default, so you won't end up with 100GB of `node_modules` across the projects on your machine. This also makes performance way better for installation, update, etc.
+- stability: the way pnpm handles workspaces (monorepo) and `node_modules` in each package is much better than yarn: you have a node_modules folder in all your packages, rather than everything in the root. The impact is that if you have a dependency that does something crazy (like Serverless that dynamically requires modules), it will work
+
+## How?
+
+Migrating from yarn to pnpm is quite straightforward:
+
+- install pnpm `npm install -g pnpm`
+- rename all your `yarn` commands to `pnpm`:
+  - `yarn` -> `pnpm install`
+  - `yarn test` -> `pnpm test`
+  - `yarn package` -> `pnpm package`
+  - `yarn deploy` -> `pnpm run deploy`
```

Comment:
I would stress this last change as it is often an issue when switching to from yarn to pnpm